packages/contracts/src.ts/index.ts

```diff
@@ -354,10 +352,10 @@ function buildEstimate(contract: Contract, fragment: FunctionFragment): Contract
     };
 }
 
-function addContractWait(contract: Contract, tx: TransactionResponse) {
+function addContractWait(contract: Contract, tx: ContractTransaction) {
```

Comment:
Is there a reason for changing this to `ContractTransaction` from `TransactionResponse`?

Reply:
Actually no, it was left from when I was trying to fix an issue that caused errors if there was no `// ts-ignore`.